flame_fire_atlas/lib/flame_fire_atlas.dart

```diff
@@ -23,52 +21,87 @@ extension FireAtlasExtensions on Game {
   }
 }
 
-abstract class Selection {
+class Selection {
   String id;
   int x;
   int y;
   int w;
   int h;
 
+  Selection({
+    required this.id,
+    required this.x,
+    required this.y,
+    required this.w,
+    required this.h,
+  });
+
+  factory Selection.fromJson(Map<String, dynamic> json) {
+    return Selection(
+      id: json['id'],
+      x: json['x'],
+      y: json['y'],
+      w: json['w'],
+      h: json['h'],
+    );
+  }
+}
+
+abstract class BaseSelection {
+  final Selection info;
```

Comment:
Maybe this class could have getters to all the info fields so that `info` doesn't need to be an intermediary in every call?

Reply:
Good idea, I wish I had thought on this before hahaha